API/src/main/java/xyz/kyngs/librelogin/api/event/events/LobbyServerChooseEvent.java

```diff
@@ -15,4 +15,26 @@
  * @see ServerChooseEvent#setServer(S)
  */
 public interface LobbyServerChooseEvent<P, S> extends ServerChooseEvent<P, S> {
+
+    /**
+     * Checks if the event was called to select a fallback server
+     *
+     * @return Whether is this event called to select fallback server
+     * */
+    boolean isFallback();
+
+    /**
+     * Abort teleportion of the player
+     *
+     * @param cancelled If set to true, this event will be aborted
+     */
+    void setCancelled(boolean cancelled);
+
+    /**
+     * Checks if the event is aborted
+     *
+     * @return Whether is the event aborted
+     * */
+    boolean isCancelled();
```

Comment:
Perhaps the canceling of an event should be defined by a super class such as CancellableEvent?

Reply:
Fixed this in [6824b00](https://github.com/kyngs/LibreLogin/pull/103/commits/6824b0025b2acbb09b6fd3bf686d77c1018cd577)